### PR TITLE
Add gentle

### PR DESCRIPTION
**Pull request by smoe**
Hello, the GENtle tool for genetic cloning still has some fan base. The code has seen a series of maintenance updates and fixes in particular from the Debian community which should now go out to a wider base of users. Please kindly help with a review of the conda recipe - it is my first ever recipe, cannot say that I truly know what I was doing. Many thanks!!

### CODE REVIEW

File: recipes/gentle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - mysql
+  run:
+    - wxwidgets
+    - tinyxml
+    - sqlite
+    - mysql
+
+about:
+  home: https://github.com/GENtle-persons/gentle-m
+  license: GPL-2+
```
Comment (bgruening) [inline recipes/gentle/meta.yaml:32]: This should be an SPDX identifier.

File: recipes/gentle/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package:
+  name: gentle
+  version: 1.9.5
+
+source:
+  git_url: https://github.com/GENtle-persons/gentle-m
```
Comment (bgruening) [inline recipes/gentle/meta.yaml:6]: can we use a release tarball here?

File: recipes/gentle/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    - {{ pin_subpackage('gentle', max_pin="x") }}
+
+requirements:
+  build:
+    - git
+    - autoconf
+    - automake
+    - make
+    - {{ compiler('cxx') }}
+    - wxwidgets
```
Comment (bgruening) [inline recipes/gentle/meta.yaml:20]: wxwidgets and all others below needs to go into a `host` section

File: recipes/gentle/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+package:
+  name: gentle
+  version: 1.9.5
+
+source:
+  git_url: https://github.com/GENtle-persons/gentle-m
+  git_depth: 1
+
+build:
```
Comment (bgruening) [inline recipes/gentle/meta.yaml:9]:
```suggestion
build:
  number: 0
```

File: recipes/gentle/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+about:
+  home: https://github.com/GENtle-persons/gentle-m
+  license: GPL-2+
+  license_file: COPYIG
+  summary: |
+    GENtle is a software for DNA and amino acid editing, database management,
+    plasmid maps, restriction and ligation, alignments, sequencer data import,
+    calculators, gel image display, PCR, and much more.
+
+extra:
```
Comment (bgruening) [inline recipes/gentle/meta.yaml:39]: more like that:

```
extra:
  recipe-maintainers:
    - old-rob
  identifiers:
    - biotools:cptac
    - doi:10.1021/acs.jproteome.0c00919
```

Comment (smoe) [reply recipes/gentle/meta.yaml:41]: Added GENtle to bio.tools :)

File: recipes/gentle/meta.yaml (hunk 3 above)
Comment (smoe) [reply recipes/gentle/meta.yaml:22]: Done.

File: recipes/gentle/meta.yaml (hunk 2 above)
Comment (smoe) [reply recipes/gentle/meta.yaml]: Released current master as 1.9.5 alpha 1 .

File: recipes/gentle/meta.yaml (hunk 1 above)
Comment (smoe) [reply recipes/gentle/meta.yaml]: Done.

**Comment by smoe** [comment]
I am tempted to give up - any idea why gcc/g++ is not found anyone?
Many thanks!

**Comment by smoe** [comment]
@bgruening Sorry for bothering you with this - configure does not find the compiler - what is the trick? On a sidenote - I was secretly hoping to see GENtle also on Windows and use conda as a vehicle. Also, can something be done to have GENtle's runtime dependency on clustalw and coils made available for the M1?

**Comment by bgruening** [comment]
> [Edit: Found it - it was a typo - bùild - I did not see that, neither did chatGPT, bard then pointed that out.]

LoL. Sorry was travelling but now I'm back in your timezone.

> On a sidenote - I was secretly hoping to see GENtle also on Windows and use conda as a vehicle. Also, can something be done to have GENtle's runtime dependency on [clustalw](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/clustalw) and [coils](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/coils) made available for the M1?

Sorry to disappoint you. If you want windows builds, you could submit your recipe to https://github.com/conda-forge/staged-recipes/

File: recipes/gentle/build.sh [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  #echo " installing it via conda"
+  #conda install gxx
+fi
+echo "which gcc:"
+which gcc || echo "gcc not in path"
+echo "which clang:"
+which clang || echo "clang not in path"
+./configure --prefix=$PREFIX CXXFLAGS="-I${PREFIX}/include"
+make
+make install
```
Comment (bgruening) [inline recipes/gentle/build.sh:29]:
```suggestion
set -ex
aclocal
libtoolize --automake --force --copy
automake --add-missing
autoconf
mkdir -p ${PREFIX}

./configure --prefix=$PREFIX CXXFLAGS="-I${PREFIX}/include"
make
make install
```

**Comment by smoe** [comment]
> > On a sidenote - I was secretly hoping to see GENtle also on Windows and use conda as a vehicle. Also, can something be done to have GENtle's runtime dependency on [clustalw](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/clustalw) and [coils](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/coils) made available for the M1?
> 
> Sorry to disappoint you. If you want windows builds, you could submit your recipe to https://github.com/conda-forge/staged-recipes/

With  a runtime dependency on clustalw, well, I feel it belongs to bioconda to avoid an inter-dependency between conda-forge and bioconda. Tempted, though :)  It is secondary and for Windows there should be a standalone version, I just look for the right kind of magic that I hope to integrate in the github release autobuild process.

What can I do to trigger builds of GENtle's runtime dependencies coils and clustalw for the M1?

**Review by bgruening — APPROVED** [review]

**Pull request merged**